app/main.py

```diff
@@ -1,6 +1,17 @@
 from typing import Callable
 
+saved_runs = {}
+
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    def arg_saver(*argv, **argc) -> callable:
```

Comment:
```suggestion
    def arg_saver(*args, **kwargs) -> callable:
```
Use generally accepted names for non-keyword and keyword arguments

Reply:
done